### ISSUE TITLE
Error on multiple PSL assignments

### ISSUE THREAD

**Issue by thvd** [forcedotcom/cli#2744]
### Summary
The docs describe you can assign multiple PSL at once using the `-n` parameter: https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_org_commands_unified.htm#cli_reference_org_assign_permsetlicense_unified

### Steps To Reproduce

1. Run this command: 
```shell
sf org assign permsetlicense -o "sfi-test-org"\
  -n "vlocity_cmt_EAndUCloudSalesPsl" \
  -n "vlocity_cmt_EAndUCloudServicePsl"
```

### Expected result
Both PSL will be assigned to the user.

### Actual result
```
Error (1): Flag --name can only be specified once
```

### System Information

```json
{
  "architecture": "darwin-x64",
  "cliVersion": "@salesforce/cli/2.29.5",
  "nodeVersion": "node-v20.11.1",
  "osVersion": "Darwin 23.2.0",
  "rootPath": "/usr/local/lib/node_modules/@salesforce/cli",
  "shell": "zsh",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 3.0.8 (core)",
    "@oclif/plugin-commands 3.1.3 (core)",
    "@oclif/plugin-help 6.0.12 (core)",
    "@oclif/plugin-not-found 3.0.10 (core)",
    "@oclif/plugin-plugins 4.2.2 (core)",
    "@oclif/plugin-search 1.0.14 (core)",
    "@oclif/plugin-update 4.1.11 (core)",
    "@oclif/plugin-version 2.0.12 (core)",
    "@oclif/plugin-warn-if-update-available 3.0.10 (core)",
    "@oclif/plugin-which 3.0.15 (core)",
    "@provartesting/provardx 0.3.2 (user)",
    "@salesforce/cli 2.29.5 (core)",
    "apex 3.0.23 (core)",
    "auth 3.3.11 (core)",
    "data 3.0.20 (core)",
    "deploy-retrieve 3.2.14 (core)",
    "dev 2.1.1 (user)",
    "info 3.0.24 (core)",
    "limits 3.1.8 (core)",
    "marketplace 1.0.22 (core)",
    "omnistudio-migration-tool 1.3.0 (user)",
    "org 3.3.12 (core)",
    "packaging 2.1.7 (core)",
    "schema 3.1.3 (core)",
    "settings 2.0.24 (core)",
    "signups 2.0.15 (user)",
    "sobject 1.1.11 (core)",
    "source 3.1.13 (core)",
    "telemetry 3.1.11 (core)",
    "templates 56.0.14 (core)",
    "trust 3.3.8 (core)",
    "user 3.2.8 (core)",
    "sfdx-hardis 4.10.2 (user)"
  ]
}

```

### Additional information
<img width="954" alt="image" src="https://github.com/forcedotcom/cli/assets/973711/ccf59f74-4a65-4aa6-8689-7f7696bba94a">

**Comment by thvd** [comment]
When i compare https://github.com/salesforcecli/plugin-user/blob/main/src/commands/org/assign/permset.ts and https://github.com/salesforcecli/plugin-user/blob/main/src/commands/org/assign/permsetlicense.ts i see a few differences which could be helpful for how to solve this.
 
Good to know: for the permset this feature is working.

**Comment by mshanemc** [comment]
I'll fix that

**Comment by jshackell-sfdc** [comment]
This issue was fixed in version 2.31.10 of Salesforce CLI.

**Comment by thvd** [comment]
@mshanemc thanks for the quick resolve on this one 👍
